libs/milvus/langchain_milvus/vectorstores/milvus.py

```diff
@@ -856,6 +853,82 @@ def add_texts(
             logger.debug("Nothing to insert, skipping.")
             return []
 
+        transposed_embeddings = [
+            [embeddings[j][i] for j in range(len(embeddings))]
+            for i in range(len(embeddings[0]))
+        ]
+        # Now:
+        # transposed_embeddings = [
+        #     [f1(a), f2(a)],
+        #     [f1(b), f2(b)],
+        #     [f1(c), f2(c)]
+        # ]
+
+        return self.add_embeddings(
+            texts=texts,
+            embeddings=transposed_embeddings,
+            metadatas=metadatas,
+            timeout=timeout,
+            batch_size=batch_size,
+            ids=ids,
+            **kwargs,
+        )
+
+    def add_embeddings(
+        self,
+        texts: List[str],
+        embeddings: Union[List[List[float]], List[List[List[float]]]],
```

Comment:
Then I've changed the signature to make it more easy to handle. This signature agrees with different implementations @Mateusz-Switala has pointed:

1. https://github.com/langchain-ai/langchain-postgres/blob/main/langchain_postgres/vectorstores.py#L747-#L803
2. https://github.com/langchain-ai/langchain/blob/master/libs/community/langchain_community/vectorstores/lantern.py#L467-#L484
3. https://github.com/langchain-ai/langchain/blob/master/libs/community/langchain_community/vectorstores/pgembedding.py#L260-#L281